README.md

```diff
@@ -2,120 +2,100 @@
 
 ## Disclaimer
 
-This is work in progress and should be considered just as a proof of concept on how to leverage ROS 2 features to use in real-time applications. Any feedback and help to improve the project is appreciated.
+This is work in progress and should be considered just as a proof of concept on how to leverage ROS
+ 2 features to use in real-time applications. Any feedback and help to improve the project is
+  appreciated.
 
-**Foxy branch is under active development. The instructions are no longer up-to-date and have to be updated. The current status can be checked here: https://github.com/ros2-realtime-demo/pendulum/issues/35**
+**Foxy branch is under active development. The current status can be checked here: 
+https://github.com/ros2-realtime-demo/pendulum/issues/35**
 
-### Build status
+## Build status
 
-| foxy | eloquent |  dashing |
-|--------- | -------- | ------- |
-| ![Build Status](https://github.com/ros2-realtime-demo/pendulum/workflows/build/badge.svg?branch=foxy) | ![Build Status](https://github.com/ros2-realtime-demo/pendulum/workflows/build/badge.svg?branch=eloquent) | ![Build Status](https://github.com/ros2-realtime-demo/pendulum/workflows/build/badge.svg?branch=dashing) |
+![Build Status](https://github.com/ros2-realtime-demo/pendulum/workflows/build/badge.svg?branch=foxy) [![License](https://img.shields.io/badge/license-Apache%202-blue)]() ![GitHub issues](https://img.shields.io/github/issues/ros2-realtime-demo/pendulum)
```

Comment:
Why this change?

Reply:
I thought that makes more sense to only show the build status for the current branch. For every release, we will be adding a new badge, also if we stop supporting an old branch and it stops passing I would prefer not to show it. I don't have a strong opinion on this, change this as you think is best.

Reply:
Fair enough.